docs/codeql/codeql-cli/analyzing-databases-with-the-codeql-cli.rst

```diff
@@ -79,17 +80,21 @@ You can also specify:
 
 - .. include:: ../reusables/threads-query-execution.rst
 
-
-.. pull-quote::
+. pull-quote::
 
    Upgrading databases
 
-   If the CodeQL queries you want to use are newer than the
-   extractor used to create the database, then you may see a message telling you
+   For databases that were creaated by CodeQL CLI v2.3.3 or earlier, you will need
+   to explicitly upgrade the database before you can run an analysis with a newer
+   version of the CodeQL CLI. If this step is necessary, then you will see a message telling you
    that your database needs to be upgraded when you run ``database analyze``.
-   You can quickly upgrade a database by running the ``database upgrade``
-   command. For more information, see ":doc:`Upgrading CodeQL databases
-   <upgrading-codeql-databases>`."
+
+   CodeQL databases are upgraded by running the following command::
```

Comment:
Add a sentence before this explaining that for databases created by newer versions of CodeQL, CodeQL will implicitly perform any necessary upgrades, and an explicit command is not required.